Formula/e/evil-helix.rb

```diff
@@ -0,0 +1,26 @@
+class EvilHelix < Formula
+  desc "Soft fork of the helix editor"
+  homepage "https://github.com/usagi-flow/evil-helix"
+  url "https://github.com/usagi-flow/evil-helix/archive/refs/tags/release-20250104.tar.gz"
+  sha256 "b03c78ea4cacd11eac8c2fb2e40c4a20e5d6d29ab151e3176876bf58c298b7b5"
+  license "MPL-2.0"
+  head "https://github.com/usagi-flow/evil-helix.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    ENV["HELIX_DEFAULT_RUNTIME"] = libexec/"runtime"
+    system "cargo", "install", "-vv", *std_cargo_args(path: "helix-term")
+    rm_r "runtime/grammars/sources/"
+    libexec.install "runtime"
+
+    bash_completion.install "contrib/completion/hx.bash" => "hx"
+    fish_completion.install "contrib/completion/hx.fish"
+    zsh_completion.install "contrib/completion/hx.zsh" => "_hx"
+  end
+
+  test do
+    assert_match "evil-helix 25.1", shell_output("#{bin}/hx --version")
```

Comment:
We need this number to be dynamically fetched from somewhere, because like this it'll fail on every update.

Reply:
I see - I was expecting when updating the formula for new releases, this would be adjusted manually. 

So until upstream fixes their versioning, can we merge without the version check?

Reply:
Looks like @daeho-ro found a solution. Thanks!